stripe/src/main/java/com/stripe/android/util/StripeTextUtils.java

```diff
@@ -81,6 +82,107 @@ public static boolean isBlank(String value) {
     }
 
     /**
+     * Returns a {@link CardBrand} corresponding to a partial card number, or {@link Card#UNKNOWN}
+     * if the card brand can't be determined from the input value.
```

Comment:
I definitely like this idea, but I think I'd like to add it as a separate function to keep things compartmentalized.